unlock-app/src/components/interface/buttons/ActionButton.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components'
+import Media from '../../../theme/media'
+
+export const ActionButton = styled.button`
+  background-color: ${props =>
+    props.disabled ? 'var(--grey)' : 'var(--green)'};
+  border: none;
+  font-size: 16px;
+  color: var(--darkgrey);
+  font-family: 'IBM Plex Sans', sans-serif;
+  border-radius: 4px;
+  cursor: ${props => (props.disabled ? 'auto' : 'pointer')};
+  outline: none;
+  transition: background-color 200ms ease;
+  & :hover {
+    background-color: var(--activegreen);
+  }
+`
+
+export const CreateLockButton = styled(ActionButton)`
+  padding: 10px;
+  align-self: end;
+  height: 48px;
+  ${Media.phone`
+    display: none;
+  `};
+`
+
+export const AccountWrapper = styled.div`
```

Comment:
This is preparatory; when we put the button adjacent to the `Account` component, this will ensure that the button goes in the correct place.